torch/csrc/distributed/rpc/init.cpp

```diff
@@ -253,6 +253,16 @@ If the future completes with an error, an exception is thrown.
           &ProcessGroupAgent::sync,
           py::call_guard<py::gil_scoped_release>());
 
+  module.def(
+      "get_current_rpc_agent",
```

Comment:
any reason this does not start with an underscore?

Reply:
I thought it's a public API. Looking thorough call sites again. I think it's only used in rpc_test.py and some rpc starter library code. makes sense to keep it conservative and add an underscore for now.